lib/ClangImporter/ClangImporter.cpp

```diff
@@ -8809,8 +8809,7 @@ swift::importer::getCxxRefConventionWithAttrs(const clang::Decl *decl) {
     if (const clang::RecordDecl *record =
             ptrType->getPointeeType()->getAsRecordDecl()) {
       return matchSwiftAttrConsideringInheritance<RC>(
```

Comment:
Not specific to this patch, but could you please remove `matchSwiftAttrConsideringInheritance`, etc. from the header and move them into the implementation file, since they aren't used anywhere outside of ClangImporter?